src/handlers/setDependencyVersionHandler.ts

```diff
@@ -108,10 +108,11 @@ async function insertDependencyManagement(pomPath: string, targetNode: ElementNo
             node.children?.find(id => id.tag === XmlTagName.GroupId && id.text === gid) &&
             node.children?.find(id => id.tag === XmlTagName.ArtifactId && id.text === aid)
         );
-        targetText = constructDependencyNode(gid, aid, version, `${baseIndent}${indent}`, indent, eol);
+        const newIndent: string = `${baseIndent}${indent}`;
+        targetText = constructDependencyNode({gid: gid, aid: aid, version: version, baseIndent: newIndent, indent: indent, eol: eol});
     } else if (targetNode.tag === XmlTagName.DependencyManagement && dependenciesNode === undefined) {
         insertPosition = currentDocument.positionAt(targetNode.contentStart);
-        targetText = constructDependenciesNode(gid, aid, version, baseIndent, indent, eol);
+        targetText = constructDependenciesNode({gid: gid, aid: aid, version: version, baseIndent: baseIndent, indent: indent, eol: eol});
```

Comment:
Can be simplified to {gid, aid, ...}.   because {gid} is short for {gid: gid} if the key name is same as variable.